record_indexer/settings.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+LOCAL_RUN = os.environ.get('INDEXER_LOCAL_RUN', False)
+DATA_SRC = os.environ.get('INDEXER_DATA_SRC', 's3')
+S3_BUCKET = os.environ.get('S3_BUCKET', False)
+
+AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID', False)
+AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY', False)
+AWS_SESSION_TOKEN = os.environ.get('AWS_SESSION_TOKEN', False)
+AWS_REGION = os.environ.get('AWS_REGION', False)
+
+if not LOCAL_RUN and DATA_SRC == 'local':
+    print(
+        "A local data source or local data destination "
+        "is only valid when the application is run locally"
```

Comment:
what's the local data destination?

Reply:
There is no local destination, only source. When I was doing development, I didn't want to keep on hitting S3 to retrieve the json files to be indexed. The json files were on my local file system, and I was adding them to an index hosted on AWS.